src/components/Editor/Preview/Popup.js

```diff
@@ -241,9 +241,6 @@ export class Popup extends Component<Props> {
 
   renderPreview() {
     const { value } = this.props;
-    if (!value) {
```

Comment:
Found this is a regression from PR #4915
In that PR, added one `if (!value)` early return[1] so `this.renderSimplePreview(value)` below for simple typed value, such "", 0, null, wouldn't be reached. 

[1] [https://github.com/devtools-html/debugger.html/blob/6d4220ecc43e4613f5db8feda6b2c25676f46259/src/components/Editor/Preview/Popup.js#L244](https://github.com/devtools-html/debugger.html/blob/6d4220ecc43e4613f5db8feda6b2c25676f46259/src/components/Editor/Preview/Popup.js#L244)

Reply:
Mind leaving a comment so it becomes obvious that there should not be an early return because this breaks things?

Reply:
@nyrosmith That helps in the future. Added